prow/plugins/releasenote/releasenote_test.go

```diff
@@ -113,10 +115,10 @@ func TestReleaseNoteComment(t *testing.T) {
 				Assignees:   []github.User{{Login: "r"}},
 			},
 		}
-		if tc.isAuthor {
+		if tc.isMember {
+			ice.Comment.User.Login = "member"
+		} else if tc.isAuthor {
 			ice.Comment.User.Login = "a"
-		} else if tc.isReviewer {
```

Comment:
I think this could be simplified by changing this to `tc.is.member` and leaving the author block alone